tools/appimage.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+# The MIT License (MIT)
+# Copyright (c) 2016 Simon Peter
+
```

Comment:
Yes, this is based on this [existing script](https://github.com/probonopd/AppImages/blob/master/LICENSE) which comes from the [probonopd/AppImages](https://github.com/probonopd/AppImages/) project which in its entirety [is MIT licensed](https://github.com/probonopd/AppImages/blob/master/LICENSE). Hence I don't think that "copyright Microsoft Corporation" is applicable here, please advise.